protocol/synthetix/contracts/interfaces/IMarketManagerModule.sol

```diff
@@ -5,82 +5,121 @@ pragma solidity ^0.8.0;
      * @dev See `IMarket`.
+     * @param marketId The id of the market from which USD will be withdrawn.
+     * @param target The address of the account that will receive the withdrawn USD.
+     * @param amount The amount of USD to be withdraw, denominated with 18 decimals of precision.
      */
-    /// @notice target withdraws amount of synths to the marketId
-    function withdrawMarketUsd(uint128 marketId, address target, uint amount) external;
+    function withdrawMarketUsd(uint128 marketId, address target, uint256 amount) external;
 
     /**
      * @notice Returns the total withdrawable USD amount for the specified market.
+     * @param marketId The id of the market whose withdrawable USD amount is being queried.
+     * @return The total amount of USD that the market could withdraw at the time of the query, denominated with 18 decimals of precision.
      */
-    function getWithdrawableUsd(uint128 marketId) external view returns (uint);
+    function getWithdrawableUsd(uint128 marketId) external view returns (uint256);
 
     /**
      * @notice Returns the net issuance of the specified market (USD withdrawn - USD deposited).
+     * @param marketId The id of the market whose net issuance is being queried.
+     * @return The net issuance of the market, denominated with 18 decimals of precision.
      */
     function getMarketNetIssuance(uint128 marketId) external view returns (int128);
 
     /**
      * @notice Returns the reported debt of the specified market.
+     * @param marketId The if of the market whose reported debt is being queried.
+     * @return The market's reported debt, denominated with 18 decimals of precision.
      */
-    /// @notice gets the total balance of the market
-    function getMarketReportedDebt(uint128 marketId) external view returns (uint);
+    function getMarketReportedDebt(uint128 marketId) external view returns (uint256);
 
     /**
      * @notice Returns the total debt of the specified market.
+     * @param marketId The id of the market whose debt is being queried.
+     * @return The total debt of the market, denominated with 18 decimals of precision.
      */
-    /// @notice gets the total balance of the market (marketIssuance + marketReportedDebt)
-    function getMarketTotalDebt(uint128 marketId) external view returns (int);
+    function getMarketTotalDebt(uint128 marketId) external view returns (int256);
 
     /**
      * @notice Returns the total collateral for the specified market.
+     * @param marketId The id of the market whose collateral is being queried.
+     * @return The market's total collateral, denominated with 18 decimals of precision.
      */
-    /// @notice gets the snxUSD value of the collateral backing this market.
-    function getMarketCollateral(uint128 marketId) external view returns (uint);
+    function getMarketCollateral(uint128 marketId) external view returns (uint256);
```

Comment:
This is a confusing name. It's more like `getMarketCredit` or `getMarketCreditCapacity`, but it doesn't include market-provided collateral. This should probably be clarified in the comments, but I think we should rename as well. We also have `getMarketCollateralAmount` in the Market Collateral Module, which adds to the confusion.

I don't have a great suggestion off the top of my head. Let's discuss further?

Reply:
Will track in a separate PR